### ISSUE TITLE
2 test failures with 6.6.9 on Alpine Linux

### ISSUE THREAD

**Issue by Ikke** [sopel-irc/sopel#1692]
When running the test suite on Alpine Linux, the following tests are failing:

```
sopel/test_tools.py:153: in test                                                                                                                                                                                                               
    assert result == output                                                                                                                                                                                                                    
E   AssertionError: assert '[IP/Host Loo...69 Google LLC' == '[IP/Host Look...69 Google LLC'                                                                                                                                                   
E     - [IP/Host Lookup] Hostname: google-public-dns-a.google.com | Location: United States | ISP: AS15169 Google LLC                                                                                                                          
E     ?                            --------------   --       ----                                                                                                                                                                              
E     + [IP/Host Lookup] Hostname: dns.google | Location: United States | ISP: AS15169 Google LLC                                                                                                                                              
_____________________________________________________________________________________________________________ test_example_duck_0 _____________________________________________________________________________________________________________
sopel/test_tools.py:153: in test                                                                                                                                                                                                               
    assert result == output                                                                                                                                                                                                                    
E   AssertionError: assert 'https?:\\/\\...el\\.chat\\/?' == 'https://githu...pel-irc/sopel'                                                                                                                                                   
E     - https?:\/\/sopel\.chat\/?                                                                                                                                                                                                              
E     + https://github.com/sopel-irc/sopel                                                                                                                                                                                                     
_____________________________________________________________________________________________________________ test_example_tr2_1 ______________________________________________________________________________________________________________
sopel/test_tools.py:155: in test                                                                                                                                                                                                               
    assert result == output                                                                                                                                                                                                                    
E   assert '"Hey" (iw to...e.google.com)' == '"Hi" (iw to e...e.google.com)'                                                                                                                                                                   
E     - "Hey" (iw to en, translate.google.com)                                                                                                                                                                                                 
E     ?   ^^                                                                                                                                                                                                                                   
E     + "Hi" (iw to en, translate.google.com)                                                                                                                                                                                                  
E     ?   ^                                                                                                                                                                                                                                    
==================================================================================================== 3 failed, 142 passed in 13.91 seconds ====================================================================================================
```

2 of the failures seem to be external factors, the last one might be an example that was not updated.

**Comment by Ikke** [comment]
Sorry, I was meant to update sopel to the last version. On 6.6.9, 2 failures remain:

```
sopel/test_tools.py:157: in test                                                                                                                                                                                                               
    assert result == output
E   AssertionError: assert 'https?:\\/\\...el\\.chat\\/?' == 'https://githu...pel-irc/sopel'
E     - https?:\/\/sopel\.chat\/?
E     + https://github.com/sopel-irc/sopel
_____________________________________________________________________________________________________________ test_example_tr2_1 ______________________________________________________________________________________________________________
sopel/test_tools.py:159: in test
    assert result == output
E   assert '"Hey" (iw to...e.google.com)' == '"Hi" (iw to e...e.google.com)'
E     - "Hey" (iw to en, translate.google.com)
E     ?   ^^
E     + "Hi" (iw to en, translate.google.com)
E     ?   ^
==================================================================================================== 2 failed, 146 passed in 10.00 seconds ====================================================================================================
```

**Comment by Exirel** [comment]
@Ikke yeah, these failure are from test using live service (like google search and translate), which can return different results than what was expected at the time of writing them.

Even if this is not particularly good, it isn't something you need to worry about. It's annoying, and in Sopel 7 (in development) and later, I'd like to remove them, and find a better way to test the code.

**Comment by dgw** [comment]
These are all fixed on `master`, so I'm not going to keep this open. Exirel is correct that we'd like to improve the testing infrastructure for external-service-based plugins, but unless we need to release a backported fix to 6.6.x there won't be an updated release in that series.

**Comment by Ikke** [comment]
Alright with me. I've patched the tests so that they work for now.
